src/GrpcProtos/capture.proto

```diff
@@ -107,6 +107,22 @@ message FunctionCall {
   repeated uint64 registers = 8;
 }
 
+// FunctionEntry and FunctionExit are emitted by user space instrumentation.
+message FunctionEntry {
```

Comment:
So are we going to switch the uprobe events to those ones also in the future?

Reply:
I think we will stick with `FunctionCall` for these. Do you see a reason to switch?

Reply:
Reason would be simplicity on the receiver.

Reply:
> So are we going to switch the uprobe events to those ones also in the future?

No, no. These events are going to be passed to `LinuxTracing` to treat them similarly to uprobes, but the uprobes themselves will remain the way they are treated now in `LinuxTracing` (with `UprobesPerfEvent` and `UretprobesPerfEvent`).
The final product will still be `FunctionCall` for both.